pom.xml

```diff
@@ -13,7 +13,6 @@
   <packaging>pom</packaging>
 
   <modules>
-    <module>snakeyaml</module>
```

Comment:
snakeyaml/{pom.xml,README.md} appear to still be present

Reply:
Yeah, the whole module should be deleted.